src/PDPVerifier.sol

```diff
@@ -339,7 +339,8 @@ contract PDPVerifier is Initializable, UUPSUpgradeable, OwnableUpgradeable {
                 if (activeCount >= offset && resultIndex < limit) {
                     tempPieces[resultIndex] = pieceCids[setId][i];
                     tempPieceIds[resultIndex] = i;
-                    tempRawSizes[resultIndex] = pieceLeafCounts[setId][i] * 32;
+                    (uint256 padding, uint8 height,) = Cids.validateCommPv2(tempPieces[resultIndex]);
+                    tempRawSizes[resultIndex] = Cids.pieceSize(padding, height);
```

Comment:
@DarkLord017 let's just break this API, I don't see any current obvious users of this function so I don't there's a huge risk here. Remove `uint256[] memory rawSizes` from the return values. It's redundant, since as you're seeing, it's in the CID which we return.

Reply:
It seems that I added this in the first place, https://github.com/FilOzone/pdp/pull/177, so I'm to blame for the naming here. I take responsibility, but also give permission to break this! It never got consumed by the SDK which was the original intention, but with this fix we can start consuming it.